Resources/string.csv

```diff
@@ -11,735 +20,734 @@
+DisableAirshipDevices,禁用飞艇设备,Disable Airship Device,Отключить устройства на Airship
+DisableSkeldAdmin,禁用管理室地图,Disable Admin,Отключить Стол Администратора
+DisableMiraHQAdmin,禁用管理室地图,Disable Admin,Отключить Стол Администратора
+DisablePolusAdmin,禁用管理室地图,Disable Admin,Отключить Стол Администратора
+DisableAirshipCockpitAdmin,禁用飞艇(驾驶舱)管理室地图,Disable Cockpit Admin,Отключить Стол Администратора В Кабине
+DisableAirshipRecordsAdmin,禁用飞艇(档案室)管理室地图,Disable Records Admin,Отключить Стол Администратора В Архиве
+DisableSkeldCamera,禁用骷髅舰监控,Disable Camera,Отключить Камеры
+DisablePolusCamera,禁用波鲁斯监控设备,Disable Camera,Отключить Камеры
+DisableAirshipCamera,禁用飞艇监控,Disable Camera,Отключить Камеры
+DisableMiraHQDoorLog,禁用米拉总部门禁记录,Disable DoorLog,Отключить Журналы
+DisablePolusVital,禁用波鲁斯生命监测装置,Disable Vital,Отключить Пульсы
+DisableAirshipVital,禁用飞艇生命监测装置,Disable Vital,Отключить Пульсы
+IgnoreConditions,忽略条件,Ignore Conditions,Игнорируют условия
+IgnoreImpostors,忽略内鬼,Ignore Impostors,Игнорируют Предатели
+IgnoreNeutrals,忽略中立阵营,Ignore Neutrals,Игнорируют Нейтралы
+IgnoreCrewmates,忽略船员,Ignore Crewmates,Игнорируют Члены Экипажа
+IgnoreAfterAnyoneDied,忽略幽灵,Ignore After Anyone Has Died,Игнорируют умершие игроки
+LightsOutSpecialSettings,停电特殊设定,Fix Lights Special Settings,Специальные настройки при отключении Света
+DisableAirshipViewingDeckLightsPanel,禁用瞭望台配电箱(飞艇地图),Disable Viewing Deck Lights Panel(Airship),Отключить починку Света на Смотровой Палубе (Airship)
+DisableAirshipGapRoomLightsPanel,禁用升降机配电箱(飞艇地图),Disable Gap Room Lights Panel(Airship),Отключить починку Света в Комнате Пролета (Airship)
+DisableAirshipCargoLightsPanel,禁用货舱配电箱(飞艇地图),Disable Cargo Lights Panel(Airship),Отключить починку Света в Грузовом Отсеке (Airship)
+RandomSpawn,随机出生点,Random Spawn,Случайный спавн
+AirshipAdditionalSpawn,额外出生点（飞艇地图）,Additional Spawn(Airship),Дополнительный спавн(Airship)
+CommsCamouflage,通信破坏时小黑人（有Bug慎开）,Camouflage During Comms,Камуфляж при Саботаже Связи
+DisableReportWhenCC,小黑人时禁止报告尸体,Disable body report in camouflage,Запретить репортить трупы при Камуфляже
+EnableDebugMode,开启调试模式,Enable Debug Mode,Включить режим отладки
+ChangeNameToRoleInfo,首次会议替换玩家名字为介绍,Show Role Descriptions to Unmodded Client,Показать описания Ролей игрокам играющие без Мода
+RoleAssigningAlgorithm,职业分配算法,Role Assigning Algorithm,Алгоритм назначения Ролей
+RoleAssigningAlgorithm.Default,默认随机算法,Default,По Умолчанию
+RoleAssigningAlgorithm.NetRandom,.NET系统随机算法,.NET System.Random,Случайный
+RoleAssigningAlgorithm.HashRandom,哈希随机算法,HashRandom,HashRandom
+RoleAssigningAlgorithm.Xorshift,Xorshift随机算法,Xorshift,Xorshift
+RoleAssigningAlgorithm.MersenneTwister,Twister随机算法,MersenneTwister,MersenneTwister
+AirShipVariableElectrical,电气室门随机关闭（飞艇地图）,Variable Electrical(AirShip),Двери в Электрощитовой меняются случайно (Airship)
+EndWhenPlayerBug,当有玩家因Bug无法游戏时终止游戏,"End the game when the player leaves the game due to an error",Завершить игру когда игрок покидает игру из-за ошибки

```

Comment:
The game ends when a player with the TOHE mod gets a critical bug, not when the player leaves the game due to the bug

Reply:
Although the whole idea of ​​ending the game when the player leaves the game due to an error is quite good, because some players get a black screen after that

Reply:
Currently no players seem to be blacked out from this option turned on